packages/block-library/src/navigation/style.scss

```diff
@@ -122,27 +126,27 @@
 }
 
 // Default / Light styles
-.wp-block-navigation-link,
-.is-style-light .wp-block-navigation-link {
-	&:not(.has-text-color) .wp-block-navigation-link__content {
+.wp-block-navigation,
+.wp-block-navigation.is-style-light {
```

Comment:
Is the specificity increase from adding `.wp-block-navigation` here necessary?

Reply:
My thinking may be wrong but I wanted to avoid `.is-style-light` affecting non-navigation blocks with the same class name should they decide to implement `light`/`dark` style variations.